driver/swift_test.go

```diff
@@ -183,6 +183,16 @@ var _ = Describe("Swift", func() {
 		Expect(semVers).To(HaveLen(1))
 		Expect(semVers[0].String()).To(Equal("2.0.10"))
 	})
+
+	It("the initial version is optional", func() {
+		driver, err := newTestSwiftDriver("", "testitem4.txt")
+		defer deleteObject("testitem3.txt")
```

Comment:
should this be testitem4.txt?

Reply:
(or the previous line be testitem3.txt :D )

Reply:
good catch!